apps/ledger-live-mobile/src/index.tsx

```diff
@@ -484,19 +479,14 @@ const getOnboardingLinkingOptions = (acceptedTermsOfUse: boolean) => ({
   },
 });
 
-const platformManifestFilterParams = {
-  private: true,
-  branches: undefined, // will override & having it to undefined makes all branches valid
```

Comment:
This is broken now because you always filter the branches in `useRemoteLiveAppContext`